community/front-end/ofe/website/ghpcfe/forms.py

```diff
@@ -307,7 +307,7 @@ def clean(self):
         cleaned_data = super().clean()
         if cleaned_data["enable_placement"] and cleaned_data[
             "machine_type"
-        ].split("-")[0] not in ["c2", "c2d", "c3"]:
+        ].split("-")[0] not in ["a2", "a3", "c2", "c3", "c2d", "c3d", "g2", "h3", "n2", "n2d"]:
```

Comment:
In Slurm we switched to checking for types that we know will not work with placement instead of checking that type is allowed. This means that new types are allowed by default and do not require immediate change to make it work. 

Example: https://github.com/GoogleCloudPlatform/slurm-gcp/blob/03805b039bc6d878ee371c1513e164a80db362b1/scripts/resume.py#L627

Something to consider.